deploy/docker-compose.yaml

```diff
@@ -0,0 +1,76 @@
+version: '3.7'
+
+# This docker-compose file brings up a full Observation Portal stack with Task Scheduler,
+# Task Worker, and Example Frontend. There is also an associated container of redis for
```

Comment:
Update comment to remove example frontend and also update instructions for modifying the env variables referencing the .env file